client/src/components/Navbar.vue

```diff
@@ -120,6 +124,14 @@ export default defineComponent({
       />
     </div>
 
+    <v-btn
+      elevation="0"
```

Comment:
```suggestion
      elevation="0"
      class="mx-2"
```
Currently the button is touching the user icon, which is visible on hover:
![image](https://user-images.githubusercontent.com/577946/163593401-829083ab-b46b-4aa4-913e-374e32f716e2.png)
A little padding would be nice